packages/ui-components/src/components/state-indicator/state-indicator.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../assets/styles/globals';
+
+.state-indicator {
+	display: flex;
+	align-items: baseline;
+
+	.color {
+		width: 8px;
+		height: 8px;
+		border-radius: 50%;
+		margin-right: $spacing-2x;
+	}
+
+	.text {
+		color: kv-color('text');
```

Comment:
Please run lint and check if this fails, I believe mixins should be called first